Source/WebCore/css/CSSPropertyInitialValues.cpp

```diff
@@ -0,0 +1,622 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Copyright (C) 2016-2022 Apple Inc. All rights reserved.
```

Comment:
I wrote all the code that was moved into this file more recently, so I think this should not have Chromium copyright and should have a newer date. Also should have the Apple Inc rather than Google Inc version of the header.